packages/xo-web/src/xo-app/dashboard/health/vdisToCoalesce.js

```diff
@@ -0,0 +1,104 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import SortedTable from 'sorted-table'
+import Tooltip from 'tooltip'
+import { Card, CardHeader, CardBlock } from 'card'
+import { Col, Row } from 'grid'
+import { getUnhealthyVdiChainsLength, VDIS_TO_COALESCE_LIMIT } from 'xo'
+import { injectState, provideState } from 'reaclette'
+import { map, size } from 'lodash'
+import { Sr, Vdi } from 'render-xo-item'
+
+const COLUMNS = [
+  {
+    itemRenderer: (srId, { unhealthyVdiChainsLengthBySr }) => (
+      <div>
+        <Sr id={srId} link />{' '}
+        {size(unhealthyVdiChainsLengthBySr[srId]) >= VDIS_TO_COALESCE_LIMIT && (
+          <Tooltip content={_('srVdisToCoalesceWarning', { limitVdis: VDIS_TO_COALESCE_LIMIT })}>
+            <span className='text-warning'>
+              <Icon icon='alarm' />
+            </span>
+          </Tooltip>
+        )}
+      </div>
+    ),
+    name: _('sr'),
+    sortCriteria: 'name_label',
+  },
+  {
+    itemRenderer: (srId, { unhealthyVdiChainsLengthBySr }) => (
+      <div>
+        {map(unhealthyVdiChainsLengthBySr[srId], (chainLength, vdiId) => (
+          <SingleLineRow key={vdiId}>
+            <Col>
+              <Vdi id={vdiId} />
+            </Col>
+            <Col>
+              <span>{_('length', { length: chainLength })}</span>
+            </Col>
+          </SingleLineRow>
+        ))}
+      </div>
+    ),
+    name: _('vdisToCoalesce'),
+  },
+]
+
+const VdisToCoalesce = decorate([
```

Comment:
Didn't you want to change this name?

https://github.com/vatesfr/xen-orchestra/pull/6120#discussion_r809855825